Week06/lesson12/homework/carousel/js/index.js

```diff
@@ -0,0 +1,28 @@
+var images = ["animal1", "animal2", "animal3", "animal4", "animal5", "animal6"];
+
+var position = 0;
+
+function imageSource(slideNumber) {
+  $("img").attr("src","./images/animals/" + images[slideNumber] + ".jpg");
+}
+
+
+function showPrevious () {
+
+  if (position > 0) {
+    position = position - 1;
+    imageSource(position);
+  }
+}
+
+function showNext () {
+
+  if (position < 5)  {
```

Comment:
Well done keeping the code from accidentally trying to move forwards or backwards to nonexistent positions in the Array (and therefore nonexistent images). Two things to consider:

1) `if (position < images.length - 1) {}` ... so that if you add more images to the array, you don't have to change the code

2) How might you go about hiding the Forwards or Back button when you've gone as far as you can go in that direction?